mne_bids/read.py

```diff
@@ -267,6 +269,10 @@ def get_head_mri_trans(bids_fname, bids_root):
     import nibabel as nib
 
     # Get the sidecar file for MRI landmarks
+    if os.sep in bids_fname:
+        warnings.warn('`bids_fname` seems to be a path. Attempting to take '
+                      'only the name WITHOUT the path ... ')
```

Comment:
why do you need a warning here? what could go wrong?

Reply:
mmh I thought about filenames that include `op.sep` ... but that's probably rare/impossible?